src/coreComponents/physicsSolvers/fluidFlow/wells/WellSolverBase.cpp

```diff
@@ -315,4 +315,44 @@ WellControls const & WellSolverBase::getWellControls( WellElementSubRegion const
   return this->getGroup< WellControls >( subRegion.getWellControlsName());
 }
 
+real64 WellSolverBase::setNextDt( real64 const & time, const real64 & lastDt, geos::DomainPartition & domain )
+{
+  real64 nextDt = PhysicsSolverBase::setNextDt( time, lastDt, domain );
+
+  forDiscretizationOnMeshTargets( domain.getMeshBodies(), [&]( string const &,
+                                                               MeshLevel & mesh,
+                                                               arrayView1d< string const > const & regionNames )
+  {
+    mesh.getElemManager().forElementSubRegions< WellElementSubRegion >( regionNames, [&]( localIndex const,
+                                                                                          WellElementSubRegion & subRegion )
+    {
+      WellControls & wellControls = getWellControls( subRegion );
```

Comment:
Instead of having `WellControls` expose all its tables, can we simply do `wellControls.setNextDtFromTable( time, nextDt );`

Reply:
done